module/modmanager/options/options.go

```diff
@@ -0,0 +1,7 @@
+// Package modmanageroptions provides Options for configuring a mod manager
+package modmanageroptions
```

Comment:
Is there a need for this to be a new package? Why can't it be part of the existing modmanager or modmaninterface?

Reply:
using the modmanager package created a cyclic dependency in the modmanager tests when i needed to create new options structs